WalletWasabi.Packager/Program.cs

```diff
@@ -208,6 +208,10 @@ private static void Publish()
 			IoHelpers.EnsureDirectoryExists(deliveryPath);
 			Console.WriteLine($"Binaries will be delivered here: {deliveryPath}");
 
+			string[] buildInfoLines = GetBuildInfoData();
+
+			CheckUncommittedGitChanges();
```

Comment:
How come this isn't `async` as well? Nothing to await inside the function?

Reply:
> Nothing to await inside CheckUncommittedGitChanges()?

Yes